### DISCUSSION TITLE
V2 dashboard logo is broken in certain browsers

### DISCUSSION THREAD

**Discussion by death2cupbots** [firefly-iii/firefly-iii#4654] [Firefly III]
So since I switched to V2 the dashboard logo has been broken on Vivaldi on my Pinebrook Pro running Manjaro ARM KDE Plasma and on Brave on my desktop running Manjaro Gnome. Firefox has been loading the image fine, and when I open the image in a new tab in both those browsers, it loads fine.

![image](https://user-images.githubusercontent.com/21091510/114401346-d2c99200-9b70-11eb-8296-43aa6ecfa404.png)

So I figured, this must be a chromium issue. So today I pop over to Windows to see if it happens in Chrome. I accidentally opened it in Brave on Windows first, and it worked there. So I switched back to Linux and cleared the cache on Brave. That did not help either. I installed Vivaldi on the desktop, that worked fine too.

This is not something that I think can be replicated. And it's not a big deal, the site works fine. I just don't understand why it's no loading on those 2 browsers but works fine everywhere else.

**Comment by JC5** [comment]
What happens when you do right-click, view image? Is it a 404?

**Reply by death2cupbots** [reply]
No, when I do view image, it loads to /v2/images/logo-hollow.png and I see the image fine.

**Comment by niklas2810** [comment]
Do you see any error messages in the "Console Tab" of your Browser's Developer Tools (accessable via Ctrl + Shift + I or F12)? If yes, what's there? Otherwise you can check out the "Network" tab as well and see what the server responded to the request for the logo.

**Reply by death2cupbots** [reply]
The console says "Refused to load the image '[mydomain]/v2/images/logo-hollow.png' because it violates the following Content Security Policy directive: "img-src data: 'strict-dynamic' 'self' *.tile.openstreetmap.org ". 'strict-dynamic' is present, so host-based allowlisting is disabled.

**Reply by JC5** [reply]
That's on me then.
